src/Database/Query/SelectQuery.php

```diff
@@ -232,6 +234,40 @@ public function distinct(ExpressionInterface|array|string|bool $on = [], bool $o
         return $this;
     }
 
+    /**
+     * Add engine-specific optimizer hint.
+     *
```

Comment:
```suggestion
     * Add engine-specific optimizer hint.
     *
     * Currently only implemented by MySQL. Hints will not be applied to queries in other drivers.
     *
```

Reply:
> Currently only implemented by MySQL

They are available for postgres too using the pg_hint_plan extension